mc-conflicts-ui/src/conflicts/models/conflict-search-params.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { ConflictSearchParams } from './conflict-search-params';
+import { Geometry } from 'geojson';
+
+const geom: Geometry = {
+  type: 'Polygon',
+  coordinates: [
+    [
+      [0, 0],
+      [0, 1],
+      [1, 1],
+      [1, 0],
+      [0, 0],
+    ],
+  ],
+};
+
+it('isDateRangeValid returns true when both dates are undefined', () => {
+  const store = ConflictSearchParams.create({});
+
+  expect(store.isDateRangeValid).toEqual(true);
+});
+
+it('isDateRangeValid returns true if only one date is defined', () => {
+  const store = ConflictSearchParams.create({ from: new Date() });
+
+  expect(store.isDateRangeValid).toEqual(true);
+});
+
+it('isDateRangeValid returns false if both dates are defined, but from > to', () => {
+  const store = ConflictSearchParams.create({
+    from: new Date(1595222000),
+    to: new Date(1595221000),
+  });
+
+  expect(store.isDateRangeValid).toBe(false);
+});
+
+it('isDateRangeValid returns false if both dates are defined, and from is before to', () => {
+  const store = ConflictSearchParams.create({
+    from: new Date(1595222000),
+    to: new Date(1595223000),
```

Comment:
I would use the same suggestion as described above.

Reply:
fixed in b2e85dc